impl/client/src/main/javascript/web/pentaho/visual/views/echarts/Funnel.js

```diff
@@ -191,7 +123,7 @@ define([
       };
 
       this._configureData(options, records);
-      this._configureLabel(options);
+      this._configureLabel(options, this._getEChartsLabel(this.model.labelsOption), "{b}: {d}%");
```

Comment:
Why was `_configureLabel` moved to the base, and its signature changed, if the Gauge chart does not use it?

I think moving it to the base class is a good idea, however. Also, would be nice to have a default `_updateAll` which would call it. It can also be called conditionally, based on the presence of the `labelsOption` property in the model. For example:
```js
if ("labelsOption" in this.model) {
  this._configureLabel(options);
}
```

In this case, the `_getEChartsLabel` method would also be moved.

I can imagine the `labelsOption` property being used on a Gauge chart.